src/Sylius/Bundle/CustomerBundle/.gitignore

```diff
@@ -6,3 +6,7 @@ composer.lock
 
 test/app/cache
 test/app/logs
+
```

Comment:
Please format it the same as for other bundles.

![image](https://github.com/Sylius/Sylius/assets/870747/f4a38118-d725-439b-afb4-867478576a86)

Reply:
fixed